### ISSUE TITLE
Local Build Error with JIT enabled: ENOENT: no such file or directory

### ISSUE THREAD

**Issue by pixleight** [nuxt-community/tailwindcss-module#307]
### Version
@nuxtjs/tailwindcss: 4.0.0
nuxt: 2.15.3
node: 14.16.0

### Reproduction Link
https://github.com/pixleight/nuxt-tailwind-jit-issue/

### Steps to reproduce
1. Clone Nuxt codesandbox at https://codesandbox.io/s/o4vn5pvp7q to local
2. Install dependencies: `yarn install`
3. Serve Nuxt: `yarn dev` OR build Nuxt: `yarn build`

### What is Expected?
Nuxt and Tailwind to build successfully.

### What is actually happening?
Build process is failing locally with the following error (example running `yarn dev`):

```bash
 ERROR  Failed to compile with 1 errors


 ERROR  in ./node_modules/@nuxtjs/tailwindcss/dist/runtime/tailwind.css 

Module build failed (from ./node_modules/@nuxt/postcss8/node_modules/postcss-loader/dist/cjs.js):
Error: ENOENT: no such file or directory, stat '/path/to/nuxt-tailwind-jit-issue/node_modules/@nuxtjs/tailwindcss/dist/runtime/tailwind.css'
    at Object.statSync (fs.js:1086:3)
    at trackModified (/path/to/nuxt-tailwind-jit-issue/node_modules/@tailwindcss/jit/src/lib/setupContext.js:188:26)
    at /path/to/nuxt-tailwind-jit-issue/node_modules/@tailwindcss/jit/src/lib/setupContext.js:653:38
    at /path/to/nuxt-tailwind-jit-issue/node_modules/@tailwindcss/jit/src/index.js:34:49
    at LazyResult.runOnRoot (/path/to/nuxt-tailwind-jit-issue/node_modules/postcss/lib/lazy-result.js:303:16)
    at LazyResult.runAsync (/path/to/nuxt-tailwind-jit-issue/node_modules/postcss/lib/lazy-result.js:355:26)
    at async Object.loader (/path/to/nuxt-tailwind-jit-issue/node_modules/@nuxt/postcss8/node_modules/postcss-loader/dist/index.js:95:14)


 @ ./node_modules/@nuxtjs/tailwindcss/dist/runtime/tailwind.css 4:14-207 15:3-20:5 16:22-215
 @ ./.nuxt/App.js
 @ ./.nuxt/index.js
 @ ./.nuxt/client.js
 @ multi ./node_modules/eventsource-polyfill/dist/browserify-eventsource.js (webpack)-hot-middleware/client.js?reload=true&timeout=30000&ansiColors=&overlayStyles=&path=%2F__webpack_hmr%2Fclient&name=client ./.nuxt/client.js
 ```

**Comment by fmctaggart** [comment]
I'd like to add that I'm getting this error as well - however it shows the file `'.../assets/css/tailwind.css'.` The error stops when I remove:
```
@tailwind base;
@tailwind components;
@tailwind utilities;' 
```
from the file.

**Comment by pixleight** [comment]
I just tried adding `./assets/css/tailwind.css`, with the `@tailwind` directives it still gives me an error.

If I remove those directives as @fmctaggart mentions, the error goes away and the build succeeds, but no styles are generated (not that I'd expect they would, without the directives for tailwind)

**Comment by pixleight** [comment]
Update: it looks like this is related to https://github.com/tailwindlabs/tailwindcss-jit/issues/22 and not specific to this module.

TL;DR: it doesn't like paths with spaces.

I moved my project to a directory without spaces anywhere in the absolute path and everything worked.

**Comment by pixleight** [comment]
Closing since it's not specific to this package.
